application/models/Token.php

```diff
@@ -61,6 +61,9 @@ abstract class Token extends Dynamic
     /** @var int Maximum token length */
     const MAX_LENGTH = 36;
 
+    /** @var int Default token length */
+    const DEFAULT_LENGTH = 15;
```

Comment:
We don't have a config for this ? I think yes, but no ....

Reply:
Would be the best, but we don't have it and I just wanted to wrap the ticket :)
The less changes, the less tests, the less destabilization

Reply:
totally right :+1: